neuralop/layers/spectral_convolution.py

```diff
@@ -132,7 +127,7 @@ def _contract_tt(x, tt_weight, separable=False):
         return tl.einsum(eq, x, *tt_weight.factors)
 
 
-def get_contract_fun(weight, implementation="reconstructed", separable=False):
```

Comment:
Why did you remove the separable arg? How will we choose the right function?

Reply:
All the factorized contraction functions have their own separable arg, which is passed at the call time for `self._contract`. In the reconstructed case, `_contract_dense_separable` is just elem-wise multiplication `x * w` as opposed to using einsum. I can add it back in